source/2020-03-20-the-ember-times-issue-140.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: The Ember Times - Issue No. 140
-author: Chris Ng, the crowd
+author: Chris Ng, Jessica Jordan, Isaac Lee, the crowd
```

Comment:
I added Jessica's name here to remind myself/whoever merges this PR that we may need to fix a merge conflict.